lib/index.test-d.ts

```diff
@@ -21,14 +21,14 @@ const upload = new tus.Upload(file, {
   },
   onProgress: (bytesSent: number, bytesTotal: number) => {
     const percentage = ((bytesSent / bytesTotal) * 100).toFixed(2)
-    console.log(bytesSent, bytesTotal, percentage + '%')
+    console.log(bytesSent, bytesTotal, `${percentage}%`)
   },
-  onChunkComplete: (chunkSize: number, bytesAccepted: number) => {},
+  onChunkComplete: (_chunkSize: number, _bytesAccepted: number) => {},
   onSuccess: () => {
     console.log('Download from %s complete', upload.url)
   },
   onError: (error: Error | DetailedError) => {
-    console.log('Failed because: ' + error)
+    console.log(`Failed because: ${error}`)
```

Comment:
Unrelated but it caught my eye
```suggestion
    console.error(`Failed because: ${error}`)
```